os/kernel/fault_manager/fault_manager.h

```diff
@@ -0,0 +1,152 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __INCLUDE_FAULT_MANAGER_H
+#define __INCLUDE_FAULT_MANAGER_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#define PATH_LEN		128		/* Application path length */
+#define FAULTMGR_STACK_SIZE	2048	/* Stack size of fault manager */
+#define FAULTMGR_PRIORITY	255	/* Fault manager priority */
+#define FAULT_MGR_MQ		"fault_queue"	/* message queue name for fault manager */
+
+/* This structure is used to store the mapping between app id and application path
+ * Its value is filled at time of registering the application for fault manager
+ */
+struct apptable {
```

Comment:
I think you'd better to change like below.
struct fm_app_table_s {
};
typedef struct fm_app_table_sfm_app_table_t;

Reply:
And please change 'app' -> 'binary' in whole files

Reply:
* I will modify the structure name.
* Why to change from app -> binary? I think fault manager monitors the applications for any fault occur. Isn't app fine for that?

Reply:
For second,
1. A binary can contain two applications. It is user choice, we don't need to force an user to split. But user can't use memory protection or other feature what we provide. That's all.
2. Internally we don't want to use an "app" word.